### ISSUE TITLE
`OrderedDict` and `Counter` being flagged when importing from `collections`

### ISSUE THREAD

**Issue by bryanforbes** [microsoft/pyright#5859]
**Describe the bug**
Given the following code:

```python
from collections import Counter, OrderedDict
```

And `deprecateTypingAliases = true`, two errors are generated:

```
error: This type is deprecated as of Python 3.9; use "collections.Counter" instead  (reportDeprecated)
error: This type is deprecated as of Python 3.9; use "collections.OrderedDict" instead (reportDeprecated)
```

**VS Code extension or command-line**
Are you running pyright as a VS Code extension, a language server in another editor, integrated into Pylance, or the command-line tool?
Command line tool and language server in neovim

Which version?
1.1.325

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.326, which I just published. It will also be included in a future release of pylance.
